bindings/python/iota_sdk/types/transaction_with_metadata.py

```diff
@@ -50,3 +50,29 @@ class TransactionWithMetadata:
     inputs = List[OutputWithMetadata]
     note: Optional[str] = None
     block_id: Optional[HexStr] = None
+
+
+@json
+@dataclass
+class CreateNativeTokenTransaction:
+    """A transaction that created a native token.
+
+    Attributes:
+        token_id: The id of the created native token.
+        transaction: The transaction that created the native token.
+    """
+    token_id: HexStr
+    transaction: TransactionWithMetadata
+
+
+@json
+@dataclass
+class CreateDelegationTransaction:
```

Comment:
Not really sure it should be in this file...

Reply:
why not?

Reply:
Name suggest it's juste for the TransactionWithMetadata type, could make it a bit more generic if you want them all together. But I'd keep TransactionWithMetadata separate tbh

Reply:
the CreateNativeTokenTransaction is there too

Reply:
and these are just wrappers

Reply:
Well it's here because you've added it, but it's fine, I'll resolve